.github/test_sdk/CMakeLists.txt

```diff
@@ -7,5 +7,5 @@ find_package(f3d REQUIRED)
 add_executable(test_sdk main.cxx)
 target_link_libraries(test_sdk f3d::libf3d)
 
-# make sure F3D SDK is compatible with C++11
-target_compile_features(test_sdk PRIVATE cxx_std_11)
+# make sure the libf3d API is compatible with C++11
+set_property(TARGET test_sdk PROPERTY CXX_STANDARD 11)
```

Comment:
I prefer `set_target_properties` function, but I don't mind if you prefer this one.